.github/workflows/ci.yml

```diff
@@ -8,13 +8,40 @@ jobs:
      matrix:
         platform: [ubuntu-18.04]
     runs-on: ${{ matrix.platform }}
+    env:
+      MEMGRAPH_HOST: "127.0.0.1"
+      MEMGRAPH_PORT: 7687
+      MEMGRAPH_SSLMODE: true
     steps:
     - uses: actions/checkout@v1
+    - name: Cache Memgraph Docker image
+      id: cache-memgraph-docker
+      uses: actions/cache@v1
+      with:
+        path: ~/memgraph
+        key: cache-memgraph-v1.1.0-docker-image
+
+    - name: Download Memgraph Docker image
+      if: steps.cache-memgraph-docker.outputs.cache-hit != 'true'
+      env:
+        MG_VERSION: '1.1.0'
+      run: |
+        mkdir ~/memgraph
+        curl -L https://memgraph.com/download/memgraph/v${MG_VERSION}/docker/memgraph-${MG_VERSION}-community-docker.tar.gz > ~/memgraph/memgraph-docker.tar.gz
+
+    - name: Load Memgraph Docker image
+      run: |
+        docker load -i ~/memgraph/memgraph-docker.tar.gz
+
+    - name: Run Memgraph instance via Docker
+      run: |
+        docker run -d -p 7687:7687 -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph memgraph
```

Comment:
It is not an issue, but you can actually dismiss the volume mount as it serves no benefit for the actions. Every action will have new volumes where a simpler command `docker run -d -p 7687:7687 memgraph` would work the same.